src/components/all-on-one-form-vc-button/all-on-one-form-vc-button.ts

```diff
@@ -43,6 +44,11 @@ export class AllOnOneFormVcButtonComponent {
     this.mc.on('press', (e) => {
       this.addDrivingFault();
     });
+
+    // Add a fault badge counter if there is a tell me fault
+    if (this.isFaultType('tellMe', 'fault')) this.vCheckButton.faultCount = 1;
+    if (this.isFaultType('tellMe', 'serious')) this.vCheckButton.serious = true;
+    if (this.isFaultType('tellMe', 'dangerous')) this.vCheckButton.dangerous = true;
```

Comment:
Just realised, we could have just moved the store subscriber in the vCheckButton component to ngOnInit() instead of having it in the constructor and it would have picked up the vehicle check state but I guess this has less impact on any other components in use.